GOVERNANCE.md

```diff
@@ -0,0 +1,140 @@
+# Project Governance
+
+## Working Group
+
+All of the Node.js Foundation websites, including this repo, are jointly governed
+by a Working Group (WG). The WG is responsible for high-level guidance of this and
+related projects which include:
+
+- [nodejs/iojs.org](https://github.com/nodejs/iojs.org) published to https://iojs.org
+- [nodejs/new.nodejs.org](https://github.com/nodejs/iojs.org) published to https://new.nodejs.org
+- [joyent/node-website](https://github.com/joyent/node-website) published to https://nodejs.org
+
+The Website WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+[README.md](./README.md#current-project-team-members).
+
+## Collaborators
+
+The [nodejs/new.nodejs.org](https://github.com/nodejs/new.nodejs.org)
+GitHub repository is maintained by the WG and additional Collaborators who are
+added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These
+individuals are identified by the WG and their addition as
+Collaborators is discussed during the weekly WG meeting.
+
+_Note:_ If you make a significant contribution and are not considered
+for commit-access log an issue or contact a WG member directly and it
+will be brought up in the next WG meeting.
+
+Modifications of the contents of the `nodejs/new.nodejs.org` repository are made on
+a collaborative basis. Anybody with a GitHub account may propose a
+modification via pull request and it will be considered by the project
+Collaborators. All pull requests must be reviewed and accepted by a
```

Comment:
This may not be worthwhile, especially if we are redeveloping parts of the site.